test/parallel/test-querystring-maxKeys-Infinity.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+const common = require('../common');
```

Comment:
Didn't our linter complain about unused variable? I thought [this](https://github.com/nodejs/node/blob/master/.eslintrc#L105-L106),

```
  ## disallow declaration of variables that are not used in the code
  no-unused-vars: [2, {"args": "none"}]
```

forbids us